src/data_types/strs.rs

```diff
@@ -52,21 +53,30 @@ pub enum FromStrWithBufError {
     BufferTooSmall,
 }
 
-/// A Latin-1 null-terminated string
+/// A null-terminated Latin-1 string.
 ///
-/// This type is largely inspired by `std::ffi::CStr`, see the documentation of
-/// `CStr` for more details on its semantics.
+/// This type is largely inspired by [core::ffi::CStr] with the exception that all characters are
+/// guaranteed to be 8 bit long. A [CStr8] can be constructed from a [core::ffi::CStr] via a
```

Comment:
`[CStr8]` -> ``[`CStr8`]``
`[core::ffi::CStr]` -> ``[`core::ffi::CStr`]``

Reply:
Ahhh. Until a few months ago I always added the backticks. Then I started omitting them as I learned, they are not required for the intra-doc link magic. 

TIL that the backticks change the appearance of the intradoc link to code formatting. Thanks! :D